week-2/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount >= mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = studentCount <= roomMaxCapacity ; // finish this statement
```

Comment:
Great work Ivina.  I think this is to find out if there is enough space in room for students and mentors(not just the students).